src/changelogs/parse_changelogs.py

```diff
@@ -1,88 +1,230 @@
             if line.startswith('[ '):
                 current_heading = line[2:-2]
-                changelog_dict[current_heading] = {default_category: []}
+                changelog_out.append(
+                    {'Description': f'<h4>{current_heading}</h4>', 'Tags': [self.default_tag]}
+                )
                 continue
 
-            # find if a resource can be assigned a changelog line
-            resource_found = False
-            for resource_key in self.resources:
-                resource = self.resources[resource_key]
-                resource_type = resource['Type']
-                resource_name = resource['Name']
-                if resource_name is None:
-                    continue
-
-                if resource_name in line:
-                    resource_found = True
-                    if resource_name not in changelog_dict[resource_type]:
-                        changelog_dict[resource_type][resource_name] = []
-
-                    # strip redundant prefix as it is already grouped under resource_name
-                    if line.startswith(f'- {resource_name}: '):
-                        line = line.replace(f'{resource_name}: ', '')
+            # parse line:
+            # replace hyphen with asterisk for bullet points on wiki
+            prefixes = ['- ', ' - ']
+            for prefix in prefixes:
+                if line.startswith(prefix):
+                    line = '* ' + line[len(prefix) :]
+            # replace -> with →
+            line = line.replace('->', '→')
 
-                    changelog_dict[resource_type][resource_name].append(line)
+            tags = self._parse_tags(current_heading, line)
 
-            if not resource_found:
-                changelog_dict[current_heading]['General'].append(line)
+            changelog_out.append({'Description': line, 'Tags': tags})
 
-        changelog_with_icons = self._embed_icons(changelog_dict)
+        changelog_with_icons = self._embed_icons(changelog_out)
         os.makedirs(self.OUTPUT_CHANGELOGS, exist_ok=True)
-        json_utils.write(self.OUTPUT_CHANGELOGS + f'/date/{version}.json', changelog_with_icons)
+        json_utils.write(self.OUTPUT_CHANGELOGS + f'/versions/{version}.json', changelog_with_icons)
         return changelog_with_icons
 
-    # mass find and replace of any resource names with the ability icon template
+    # Parse a given line for assignable tags
+    def _parse_tags(self, current_heading, line):
+        tags = []
+
+        # find if a resource can be assigned a changelog line
+        for resource_key in self.resources:
+            resource = self.resources[resource_key]
+            resource_type = resource['Type']
+            resource_name = resource['Name']
+
+            if resource_name is None:
+                continue
+
+            # Skip disabled items
+            if resource_type == 'Item' and resource.get('Disabled', False):
```

Comment:
Not sure about skipping these. eg. Soul Rebirth was recently removed, if we ignored disabled items, then we would have missed that tag

Reply:
I would agree, I didn't have this initially. The reason I restricted it to items only is specifically because there are disabled items with what are I presume placeholder names like `Fire Rate` and `Health Regen`. This caused them to be flagged as Items in tons of places where it was obviously referencing a stat. 

Looking through item data now, it seems it's actually just these two. Instead, I'll replace this condition with just not being either of these two specific names.

Reply:
Ohh interesting. Hopefully no more terrible item names like that in future